### ISSUE TITLE
Cucumber task not registered with Leiningen

### ISSUE THREAD

**Issue by undees** [nilswloka/lein-cucumber#14]
Added lein-cucumber to `dev-dependencies`, per the README:

```
(defproject foo "1.0.0"
  :description "A demo project"
  :dependencies [[org.clojure/clojure "1.3.0"]]
  :dev-dependencies [[nilswloka/lein-cucumber "0.2.0"]])
```

Still couldn't run the cucumber task:

```
$ lein2 cucumber
That's not a task. Use "lein help" to list all tasks.
```

Is there another step I'm missing?

**Comment by nilswloka** [comment]
Have a look at the1.0 branch for lein2 compatibility. I'll try to release a new version of lein-cucumber a soon as possible.

**Comment by nilswloka** [comment]
Released 1.0.0 of lein-cucumber to clojars.

**Comment by undees** [comment]
I'm able to get lein2 to run Cucumber features now; thanks!
